assets/scss/app.scss

```diff
@@ -1,3 +1,6 @@
-$icon-font-path: '/fonts/'; /* use fonts from priv/static/fonts/ */
-@import 'node_modules/font-awesome/scss/font-awesome';
-@import 'bootstrap'; /* imports node_modules/bootstrap-sass/assets/stylesheets/_bootstrap.scss */
+@import "./scss/bulmaswatch/_variables.scss";
+@import "./node_modules/bulma/bulma.sass";
+@import "./scss/bulmaswatch/_overrides.scss";
+@import "forum.css";
```

Comment:
Prefer single quoted strings

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#stringquotes)